org.jellyfin.JellyfinServer.yml

```diff
@@ -0,0 +1,377 @@
+id: org.jellyfin.JellyfinServer
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+  - org.freedesktop.Sdk.Extension.node14
+  - org.freedesktop.Sdk.Extension.llvm15
+separate-locales: false
+command: jellyfin-launcher.sh
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /man
+  - /share/cmake
+  - /share/doc
+  - /share/ffmpeg
+  - /share/info
+  - /share/man
+  - /share/pkgconfig
+
+finish-args:
+  - --device=dri
+  - --filesystem=/media:ro
+  - --filesystem=/mnt:ro
+  - --filesystem=/run/media:ro
+  - --share=network
+  # - --device=all # Needed for v4l2m2m
+  # - --filesystem=home:ro
+  # - --filesystem=host:ro
+  # - --filesystem=xdg-download:ro
+  # - --filesystem=xdg-music:ro
+  # - --filesystem=xdg-pictures:ro
+  # - --filesystem=xdg-videos:ro
+
+modules:
```

Comment:
I'd recommend setting up x-checker-data for the sources. It won't update your nuget-generated-sources yet, but it'll handle the heavy stuff for you.

Reply:
Added x-checker-data for the sources.